src/configurations/destinations/mp/ui-config.json

```diff
@@ -188,6 +188,22 @@
                     "regex": "^(.{0,100})$",
                     "regexErrorMessage": "Invalid Property Name",
                     "placeholder": "e.g: Cart-Value"
+                  },
```

Comment:
As setOnceProperties is at defaultConfig level, this configuration will be same across all connection modes and that was the previous behaviour too ?

Reply:
Yes KC